Monster.m

```diff
@@ -50,10 +50,14 @@
 			% Configure logs
 			setpref('monsterLog', 'logToFile', Config.Logs.logToFile);
 			setpref('monsterLog', 'logFile', Config.Logs.defaultLogName);
+            
+            % Adapt to current scenario
+            monsterLog('(MONSTER - setupSimulation) applying scnario parameters', 'NFO');
+            ApplyScenario(Config);
```

Comment:
I agree. The reasoning was that MonsterConfig did not need another 200 lines, but it could just call the ApplyScenario if a scenario is chosen, but then the scenario would have to be set directly in MonsterConfig.